src/index.html

```diff
@@ -1,14 +1,56 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#Home">
+          <img
+            src="./images/logo.png"
+            alt="logo"
+            class="logo__img"
+          >
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="list">
+          <li class="list__item">
+            <a href="#" class="list__link is-active"> Apple </a>
```

Comment:
```suggestion
            <a href="#apple" class="list__link is-active"> Apple </a>
```
add more specific link names